packages/react-native/src/migrations/update-14-6-0/update-jest-config.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  Tree,
+  formatFiles,
+  getProjects,
+  ProjectConfiguration,
+  logger,
+  stripIndents,
+} from '@nrwl/devkit';
+
+/**
+ * Remove the testRunner
+ */
+export default async function update(tree: Tree) {
```

Comment:
Is this the same as the other migration?

Reply:
removed duplicate